api/app.js

```diff
@@ -90,6 +92,41 @@ module.exports = function(configFilePath) {
     res.status(HttpStatus.BAD_REQUEST).json(errorResponse);
   });
 
+  var publicApp = express();
+  publicApp.use(bodyParser.json());
+  publicApp.use(bodyParser.urlencoded({ extended: false }));
+  publicApp.use(function(req, res, next){
+    oauth.checkUserAuthorization(req,function(error,isSpaceDeveloper){
+      if(error){
+        res.status(HttpStatus.BAD_REQUEST).send({});
```

Comment:
According to "checkUserAuthorization",  the possible error could be 
1. error when connect to cc or uaa
2. invalid user token

It is incorrect to return 400 bad request in these condition